.github/workflows/release-snapshot.yml

```diff
@@ -0,0 +1,38 @@
+name: Release snapshot
+
+# Releases a snapshot release when new commits merge to develop
+# This ensures the release process is working as expected as well as always gives us a release
+# we should move to a single branch in near future
+on:
+  workflow_dispatch:
+  push:
+    branches:
+      - develop
```

Comment:
This replaces our canary releases and this will happen automatically everytime we merge to develop